python/mxnet/metric.py

```diff
@@ -81,7 +81,7 @@ def update(self, labels, preds):
         assert len(labels) == len(preds)
         for pred, label in zip(preds, labels):
             self.sum_metric += self._feval(label, pred)
-        self.num_inst += 1
+            self.num_inst += 1
```

Comment:
LGTM, but why the evaluation wasn't done in batch way? i.e. _feval takes labels and preds